### DISCUSSION TITLE
Ender 3 V2 Neo (Ender3V2-422-BLTUBL-MPC) Reverse Rotary Encoder?

### DISCUSSION THREAD

**Discussion by LunarstarFurry** [mriscoc/Ender3V2S1#1390] [Q&A]
So i wonder if anyone knows how to reverse the rotary encoder? Recently replace the rotary encoder on my panel and now everything is rotating in reverse(

**Comment by mriscoc** [comment]
Enable the corresponding settings here:
https://github.com/mriscoc/Ender3V2S1/blob/4f65c631d46ecbf4a2944716aa8c7a169e7d7432/Marlin/Configuration.h#L2687-L2718
